src/doc/book/associated-types.md

```diff
@@ -11,7 +11,7 @@ this:
 trait Graph<N, E> {
     fn has_edge(&self, &N, &N) -> bool;
     fn edges(&self, &N) -> Vec<E>;
-    // etc
+    // Etc.
```

Comment:
"etc." and "..." both indicate a continuation of a list, so it seems redundant to have both.